app/lib/actions/katello/repository/import_upload.rb

```diff
@@ -2,17 +2,27 @@ module Actions
   module Katello
     module Repository
       class ImportUpload < Actions::EntryAction
-        def plan(repository, upload_id, options = {})
+        def plan(repository, upload_ids, options = {})
           action_subject(repository)
-          unit_key = options.fetch(:unit_Key, {})
+          unit_keys = options.fetch(:unit_keys, {})
           generate_metadata = options.fetch(:generate_metadata, true)
-          import_upload = plan_action(Pulp::Repository::ImportUpload,
-                                      pulp_id: repository.pulp_id,
-                                      unit_type_id: repository.unit_type_id,
-                                      unit_key: unit_key,
-                                      upload_id: upload_id)
+          sync_capsule = options.fetch(:sync_capsule, true)
+          sequence do
+            concurrence do
+              upload_ids.zip(unit_keys) do |upload_id, unit_key|
```

Comment:
@akofink you win the 'rarely used ruby methods of the day' award for February 16, 2017!  CONGRATS 🎉 🎈 🍰